### DISCUSSION TITLE
Que tal team, poniéndome al día! buscando equipo hackathon.

### DISCUSSION THREAD

**Discussion by stgosm** [Launch-X-Latam/MisionFrontEnd#499] [Team Posts]
Que tal team!, poniéndome al día estoy buscando algún equipo para poder contribuir y crecer. Tengo conocimiento por ambos lados tanto en el back como front, en front tanto HTML, CSS, React y Javascript. En este caso para la misión back de lo que veo día a día es con SQL y C#. 
Conozco Mongo y también un poco de Python.

**Comment by FaustoPaFl** [comment]
Me apunto, deberías crear un chat en Teams así podríamos estar en contacto.
No tengo la experiencia que tienes en back-end pero soy bastante bueno en front.

**Comment by alunator** [comment]
Hola! @CristopherSandoval y yo buscamos integrantes para nuestro equipo ^^

**Comment by CristopherSandoval** [comment]
Hola Santiago! Cómo te va? Yo formé un equipo con una compañera :) Somos 2 y nos faltan 3. Te unes?

**Comment by gumodi35** [comment]
Hola! Pueden incluirme en su equipo, igual estoy buscando! 🖥️

**Comment by edison-sc** [comment]
Hola que tal, estoy buscando grupo, si aun hay espacio

**Comment by CristopherSandoval** [comment]
@FaustoPaFl Síi, mándame tu usuario de github para formar un equipo :)

**Comment by AleBT75** [comment]
Se puede aún? Espero poder unirme a su grupo

**Comment by stgosm** [comment]
Hola @CristopherSandoval, muchas gracias por la invitación y si claro que si me les uno

**Comment by giankpetrov** [comment]
lx09705@innovaccion.mx @stgosm puedes ver mi repositorio si das click en mi perfil y contactarme

**Comment by FaustoPaFl** [comment]
@CristopherSandoval  de git o de teams?
en teams soy lx03834@innovaccion.mx y pues aqui es FaustoPaFl

**Comment by Wkinson** [comment]
Muchas Gracias, es Wkinson

El 8 nov. 2022 12:16 p. m., Cristopher Sandoval ***@***.***> escribió:

@FaustoPaFl<https://github.com/FaustoPaFl> Síi, mándame tu usuario de github para formar un equipo :)

**Comment by stgosm** [comment]
@CristopherSandoval mi user de github es stgosm y de teams es lx03960@innovaccion.mx

**Comment by HoracioTorrez** [comment]
Hola si aún tienen espacio y andan en busca de un viejo de 39 años completamente nuevo en este mundo del desarrollo estaria feliz de que me aceptaran mi correo es lx15713@innovaccion.mx y mi user de github es HoracioTorrez . Gracias

**Comment by Annamalag** [comment]
Hola esté es mí nro +543855706597 mándame un WhatsApp a nosotros nos falta
un integrante

**Comment by DyFrabian** [comment]
Crees qe pueda unirme yo , Trabajo en el dia , pero en laa noches tengo
tiempo para lo qe sea

**Comment by safraZX** [comment]
Hola, yo tambien estoy buscando equipo, y me estoy poniendo al dia, tengo conocimientos en html css y js, ta,bien manejo c++ mas desarrollo de desktop, manejo backend en JS con node
________________________________
De: Santiago Serrano Márquez ***@***.***>
Enviado: martes, 8 de noviembre de 2022 12:56 p. m.
Cc: Subscribed ***@***.***>
Asunto: [Launch-X-Latam/mision-backend] Que tal team, poniéndome al día estoy buscando algún equipo para que pueda contribuir. Tengo conocimiento por ambos lados tanto en el back como front, en este caso para la misión back parte del back de lo que veo día a d...


Que tal team!, poniéndome al día estoy buscando algún equipo para poder contribuir y crecer. Tengo conocimiento por ambos lados tanto en el back como front, en front tanto HTML, CSS, React y Javascript. En este caso para la misión back de lo que veo día a día es con SQL y C#.
Conozco Mongo y también un poco de Python.

**Comment by LaBalaElmo** [comment]
Buenas, ando también buscando equipo y apoyar con lo que pueda, dejo mi correo por si a alguien le falta uno: lx10061@innovaccion.mx

**Comment by Wkinson** [comment]
Hola Cristopher como estas? Espero que estes bien, quería saber si estoy en tu equipo y quería comentarte de algunos preparativo para el hackathon o para que nos vaya bien en los desafíos.

**Comment by AlvaroFierro** [comment]
Hola necesitan alguien mas en su equipo, porque aun no encuentro :(

**Comment by Hernanjas400** [comment]
¡¡¡¡¡Hola a todos libérense de tanta ansiedad para armar Equipo!!!!! todavía no hay instrucciones concretas y si te toca solo será momento de aplicar lo aprendido.


Con tranquilidad quien no tenga Equipo me manda un correo a ***@***.*** Recibiré correos hasta 16/11/2022

Abrazo !!!!






Hernan Costanzi


________________________________
De: Alvaro ***@***.***>
Enviado: lunes, 14 de noviembre de 2022 01:47
Cc: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Que tal team, poniéndome al día! buscando equipo hackathon. (#28)


Hola necesitan alguien mas en su equipo, porque aun no encuentro :(
